### PR TITLE
Initiating release of 0.13.2

### PR DESCRIPTION
**Pull request by maayanf24**
Initiating release of 0.13.2

### CODE REVIEW

**Comment by skitt** [comment]
It seems we need https://github.com/submariner-io/shipyard/pull/1051 first; I see it was added to https://github.com/submariner-io/releases/issues/525.

File: releases/v0.13.2.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.2
+name: 0.13.2
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: 0c796bb306ee64208f6e0fc57de82cf8eee08559
```
Comment (mkolesnik) [inline releases/v0.13.2.yaml]:
```suggestion
  shipyard: 020c5e4622b747bfc99cec6d45da0ed4c81d11ea
```

**Review by mkolesnik — CHANGES_REQUESTED** [review]

**Review by mkolesnik — APPROVED** [review]

File: releases/v0.13.2.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+version: v0.13.2
+name: 0.13.2
+branch: release-0.13
+status: shipyard
+components:
+  shipyard: 020c5e4622b747bfc99cec6d45da0ed4c81d11ea
```
Comment (dfarrell07) [inline releases/v0.13.2.yaml:7]: https://github.com/submariner-io/shipyard/commits/release-0.13 :+1:

**Review by dfarrell07 — APPROVED** [review]

**Comment by dfarrell07** [comment]
[E2E (globalnet)](https://github.com/submariner-io/releases/actions/runs/3532197763/jobs/5926258424#logs) failed, re-running.

**Comment by skitt** [comment]
The DCO is missing in one commit, please rebase and squash the commits.

**Pull request merged**